### PR TITLE
open-sir: migrate scripts to package

### PR DESCRIPTION
**Pull request by jia200x**
This PR migrates the current open-sir scripts to a standard `setuptools` python package.
Now it's possible to run from the repo:

```
pip install .
```

And opensir is installed in the system.

This PR changes some stuff:
- The cli command is now called "opensir-cli"
- All modules should be imported using the opensir prefix (e.g `from opensir.models import SIR`)

This should work in all OSs (Windows, Mac-OS, Linux).

As soon as we release the 1.0.0 version, I will put this package in the "PyPi" library. That means any user can install open-sir with `pip install opensir` without cloning the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,51 +11,28 @@ The current stage of the software is *Alpha*.
...
-Next, activate the Pipenv shell:
-```
-pipenv shell
-```
-You can run the following command to check that the installation succeeded.
-```
-pipenv run start -p '[0.95,0.38]' -i '[341555,445,0]' -t 6
-```
-
+* Python >= 2.7
```
Comment (sasalatart) [inline README.md:16]: Python 2.7 is deprecated and I think we should not promote its usage. If we are going to support python versions over a certain number, I think they should be at least 3.x

What do you think?

File: README.md [hunk 2]
```diff
@@ -11,51 +11,28 @@ The current stage of the software is *Alpha*.
...
+## Installation
+Open-SIR and all its dependencies can be installed from the repository using
+`pip`:
 
-Build the Sphinx documentation out of the open-sir Pipenv environment.
 ```
-pipenv run doc
+git clone https://github.com/open-sir/open-sir.git
+cd open-sir
+sudo pip install .
```
Comment (sasalatart) [inline README.md:28]: Do we need `sudo` here?

File: README.md [hunk 3]
```diff
@@ -100,6 +77,50 @@ And learn how the API can be used to:
...
+
+Open-SIR Development Environment uses
+[Pipenv](https://pipenv.pypa.io/en/latest/) to automatically create a virtual
+environment and manage python packages. The python packages required by
+Open-SIR are listed in the [Pipfile](Pipfile).
+
+After cloning the repository, change the current directory to the repository
+via `cd open-sir` and automatically install the environment from the Pipfile
+using Pipenv:
+```
```
Comment (sasalatart) [inline README.md:90]: GitHub suggestions get confused when I suggest adding an extra line here, so I can't suggest it that way haha. Can you please add a blank line between scripts and what surrounds them please?

File: README.md [hunk 4]
```diff
@@ -100,6 +77,50 @@ And learn how the API can be used to:
...
+pipenv run test
+```
+
+The documentation is placed under `_build/html`.
+
+### Coding style tests
+
+This project uses [Pylint](https://www.pylint.org/) and [Black 19.10b0](https://black.readthedocs.io/en/stable/) to ensure consistent coding practices, and enforced by CircleCI. Non-default parameters are available in [.pylintrc](.pylintrc).
+
```
Comment (sasalatart) [inline README.md:121]: There are too many line skips here

**Comment by gwenzel** [comment]
image
Hooray !

File: README.md (hunk 1 above)
Comment (jia200x) [reply README.md:16]: you are right. Let's remove Python 2.7 from there.
Python >= 3.2 should be ok

File: README.md (hunk 2 above)
Comment (jia200x) [reply README.md:28]: actually not. I will remove it

File: README.md (hunk 3 above)
Comment (jia200x) [reply README.md:90]: sure!

**Comment by jia200x** [comment]
I added some fixups. Let me know if approved, so I can squash and rebase :)

**Comment by sasalatart** [comment]
> I added some fixups. Let me know if approved, so I can squash and rebase :)

I don't see the changes, did you forget to push?

File: README.md (hunk 1 above)
Comment (sasalatart) [reply README.md:16]: Cool 💪

**Comment by jia200x** [comment]
I forgot to force push (I amended one commit). Now it should be OK

**Comment by sasalatart** [comment]
> I forgot to force push (I amended one commit). Now it should be OK

Looks good! Ready to merge imo

**Comment by jia200x** [comment]
thank guys!

**Pull request merged**